ui/app/templates/services/index.hbs

```diff
@@ -0,0 +1,26 @@
+{{page-title "Services"}}
+<section class="section">
+	<ListTable
+		@source={{this.model.services}}
+		as |t|
+	>
+		<t.head>
+			<th>
+				Name
+			</th>
+			<th>
+				Tags
+			</th>
+		</t.head>
+		<t.body as |row|>
+			<tr data-test-service-row>
+				<td>{{row.model.name}}</td>
+				<td>
+					{{#each row.model.tags as |tag|}}
+						<span class="tag">{{tag}}</span>
+					{{/each}}
+				</td>
+			</tr>
+		</t.body>
+	</ListTable>
+</section>
```

Comment:
`question`: Is this a duplicate template? I see we have nearly the same template at `templates/services/index`? Why are we adding a duplicate of this to `templates/components/services.hbs`?